apps/inntektsmelding-service/src/main/java/no/nav/registre/testnav/inntektsmeldingservice/repository/model/InntektsmeldingModel.java

```diff
@@ -28,17 +30,30 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @EntityListeners(AuditingEntityListener.class)
-@EqualsAndHashCode
 @ToString
 public class InntektsmeldingModel {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequence_generator")
+    @SequenceGenerator(name = "sequence_generator", sequenceName = "hibernate_sequence", allocationSize = 1)
     @Column(name = "id")
     private Long id;
 
     @Temporal(TemporalType.TIMESTAMP)
     @Column(name = "created_at", nullable = false, updatable = false)
     @CreatedDate
     private Date createdAt;
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Hvis hensikten er å ekskludere createdAt kan du oppnå det samme ved å beholde lombok @EqualsAndHashCode og annotere createdAt med ExcludeEqualsAndHash, i stedet for denne eksplisitte koden.